routes/transaction.go

```diff
@@ -98,6 +98,37 @@ func (fes *APIServer) SubmitTransaction(ww http.ResponseWriter, req *http.Reques
 		return
 	}
 
+	// If this is a creator coin transaction and this node is configured for Jumio, we update global state user metadata to say user has purchased CC.
+	if txn.TxnMeta.GetTxnType() == lib.TxnTypeCreatorCoin && txn.TxnMeta.(*lib.CreatorCoinMetadataa).OperationType == lib.CreatorCoinOperationTypeBuy {
```

Comment:
I don't see the check for jumio or twilio?

Reply:
yep - I'll update the comment.